src/rpc/core_rpc_server.cpp

```diff
@@ -1990,6 +1990,35 @@ namespace cryptonote
     return true;
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  bool core_rpc_server::on_get_quorum_list(const COMMAND_RPC_GET_QUORUM_LIST::request& req, COMMAND_RPC_GET_QUORUM_LIST::response& res)
+  {
+    PERF_TIMER(on_get_quorum_list);
+    bool r;
+
+    if (use_bootstrap_daemon_if_necessary<COMMAND_RPC_GET_QUORUM_LIST>(invoke_http_mode::JON, "/get_quorum_list", req, res, r))
+    {
+      return r;
+    }
+
+    std::array<std::string, 10> quorum_array;
+    r = m_core.get_quorum_list_for_height(req.height, quorum_array);
+    if (r)
+    {
+      res.status = CORE_RPC_STATUS_OK;
+
+      // TODO: Not ideal to re-copy from std::array into std::vector, but there's no KV_SERIALIZE for arrays or primitive arrays[]
+      res.quorum_list.reserve(quorum_array.max_size());
+      for (auto it = quorum_array.begin(); it != quorum_array.end(); it++)
+        res.quorum_list.push_back(*it);
+    }
+    else
+    {
+      res.status = "Block height is too height";
```

Comment:
This may fail for other reasons, possibly check the height independently from the request. Typo for "too high"

Reply:
(Personally I would just put a vague "get_quorum_list_for_height failed" here)